src-built-in/components/workspaceManagementMenu/src/stores/workspaceManagementMenuStore.js

```diff
@@ -350,11 +363,12 @@ Actions = {
 		 * Actually perform the switch. Happens after we ask the user what they want.
 		 *
 		 */
-		function switchWorkspace() {
+		function switchWorkspace(callback) {
```

Comment:
Why is the callback needing to be introduced?

If it is required, JSDoc needs updating.

Reply:
This change fixes a different bug that was indirectly causing my fix to not work.  switchWorkspace is an internal function passed into async.waterfall(), so it has to have a callback.  Without the callback async.waterfall never finished and the corresponding "done" function (Actions.onAsyncComplete) was never called.  So this change just fixes preexisting code generally unrelated to my change.

Reply:
Ok, just add a JSDoc for it and that's reasonable. Just making sure since it is changing the signature of the function.

Reply:
done